config/manager/deployment/pdb.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: pdb
+spec:
+  minAvailable: 75%
```

Comment:
Just to double check, PDB does not interfere with the deployment's rolling upgrade strategy settings, right?
At least that's how I iunderstood it: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets 
>Pods which are deleted or unavailable due to a rolling upgrade to an application do count against the disruption budget, >but workload resources (such as Deployment and StatefulSet) are not limited by PDBs when doing rolling upgrades. >Instead, the handling of failures during application updates is configured in the spec for the specific workload resource.